### ISSUE TITLE
Porting algorithm - Multinomial NB

### ISSUE THREAD

**Issue by writetokavitha** [dotnet/machinelearning#3844]
@codemzs Not finding multinomial NB as part of the ML.Net algorithms which we use for intent detection.

From nb sci kit learn libraries:
https://scikit-learn.org/stable/modules/generated/sklearn.naive_bayes.MultinomialNB.html

clfnb = Pipeline([('vect', CountVectorizer(tokenizer=None, lowercase=True, stop_words=None, ngram_range=(1,2))), 
                  ('tfidf', TfidfTransformer(use_idf = True)),
                  ('clf', MultinomialNB()),])
clfnb.fit(df_train, df_label)

**Comment by yaeldekel** [comment]
Hi @writetokavitha,
Thank you for raising this issue. ML.NET has a multi-class Naive Bayes trainer, but it only trains on binary feature values.
I will label this feature request as an enhancement, and we will prioritize this feature.
